features/step_definitions/helpers.rb

```diff
@@ -79,11 +79,11 @@ def initialize_work_package_params(project, type = Type.find(:first), parent = n
   params
 end
 
-def initialize_impediment_params(project, sprint, user = User.find(:first))
+def initialize_impediment_params(project, sprint, user = User.first)
   params = HashWithIndifferentAccess.new(Task.new.attributes)
   params['type_id'] = Task.type
   params['fixed_version_id'] = sprint.id
-  params['status_id'] = Status.find(:first).id
+  params['status_id'] = Status.first.id
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.